3rdParty/rocksdb/v5.16.X/utilities/transactions/pessimistic_transaction.cc

```diff
@@ -125,7 +125,8 @@ bool PessimisticTransaction::IsExpired() const {
 WriteCommittedTxn::WriteCommittedTxn(TransactionDB* txn_db,
                                      const WriteOptions& write_options,
                                      const TransactionOptions& txn_options)
-    : PessimisticTransaction(txn_db, write_options, txn_options){};
+    : PessimisticTransaction(txn_db, write_options, txn_options),
+      _commited_seq_nr(0) {};
```

Comment:
I think they won't like member variable names like this...
```suggestion
      commited_seq_nr_(0) {};
```

Reply:
doesn't really matter we need to redo this patch for upstream

Reply:
Ok then. We need to make sure however the stuff somehow gets upstream, and when it is, we need to pull it back from there and remove our local adjustments.
Let's see if it gets accepted.